### PR TITLE
[ENH] Adding Contributors and updating contributions

### PR DESCRIPTION
**Pull request by franklin-feingold**

### CODE REVIEW

**Comment by franklin-feingold** [comment]
I DM'ed with both Pradeep and Fidel to confirm the contribution credit

**Comment by effigies** [comment]
I wonder if it makes more sense to make a wiki page where people continue to add themselves, and then we just import that page in the release process.

**Comment by PeerHerholz** [comment]
Sorry I'm late to the party, but being the selfish person I am, I wanted to add my name to the contributors. Before creating another PR for that: could you (@franklin-feingold) maybe add me as well? Sorry again. 

@miykael: you're also not listed yet. Wanna be included?

**Comment by effigies** [comment]
@PeerHerholz @miykael I'd suggest just adding yourself with the emojis you want using the suggest changes feature, and then @franklin-feingold can sort after committing.

**Comment by effigies** [comment]
I got some thumbs up... https://github.com/bids-standard/bids-specification/wiki/Contributors

If this seems like a good idea, we should add it to the contribution guide.

File: src/99-appendices/01-contributors.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ The following individuals have contributed to the Brain Imaging Data Structure
 ecosystem (in alphabetical order). If you contributed to the BIDS ecosystem and
 your name is not listed, please add it.
 
+-   Fidel Alfaro Almagro 💬📖💡🔌
 -   Stefan Appelhoff 📖💬🤔🐛💡💻👀⚠️📢
```
Comment (sappelhoff) [inline src/99-appendices/01-contributors.md]:
```suggestion
-   Stefan Appelhoff 📖💬🤔🐛💡💻👀⚠️📢✅🔧🔌📝
```

**Comment by raamana** [comment]
Thanks Franklin, appreciate it.

PS: I also admit my selfish- and shamelessness 😀

File: src/99-appendices/01-contributors.md [hunk 2]
```diff
@@ -101,6 +102,7 @@ your name is not listed, please add it.
 -   Russell A. Poldrack 📖🔍📢
 -   Jean-Baptiste Poline 📖📢🤔🎨
 -   Vasudev Raguram 💻🎨📖🔧
+-   Pradeep Reddy Raamana 💻🔧
 -   Ariel Rokem 📖
 -   Matt Sanderson 📖💻
```
Comment (miykael) [inline src/99-appendices/01-contributors.md]: Can you please add me as well? Thanks!
```suggestion
-   Matt Sanderson 📖💻
-   Michael P. Notter 💬📝✅📢📖
```

File: src/99-appendices/01-contributors.md [hunk 3]
```diff
@@ -48,7 +49,7 @@ your name is not listed, please add it.
 -   Eric A. Earl 🤔
 -   Anders Eklund 📖📢💻
 -   Oscar Esteban 📖🔧🤔💬💻
--   Franklin W. Feingold 📋📝✅
+-   Franklin W. Feingold 📋📝✅💬🤔
 -   Guillaume Flandin 📖💻
 -   Remi Gau 📖💻
 -   Satrajit S. Ghosh 📖💻
```
Comment (PeerHerholz) [inline src/99-appendices/01-contributors.md]:
```suggestion
-   Satrajit S. Ghosh 📖💻
- Peer Herholz 💬📖👀🔧✅📢
```

**Comment by effigies** [comment]
@franklin-feingold I updated the wiki with all of the current suggestions, if you just want to copy-paste that in.

**Comment by franklin-feingold** [comment]
Thank you @effigies for putting together the contributor wiki! I noticed for this wiki that if they are not signed in to GitHub (or perhaps an admin on the repo) they are not able to edit. I like the idea of having a dynamic page that can be copied and pasted in before a release.

File: src/99-appendices/01-contributors.md (hunk 2 above)
Comment (franklin-feingold) [reply src/99-appendices/01-contributors.md]: all set, thank you for adding in!

File: src/99-appendices/01-contributors.md (hunk 3 above)
Comment (franklin-feingold) [reply src/99-appendices/01-contributors.md]: all set, thank you for adding in!

File: src/99-appendices/01-contributors.md (hunk 1 above)
Comment (franklin-feingold) [reply src/99-appendices/01-contributors.md]: all set, thank you for updating the credit!

**Comment by sappelhoff** [comment]
> I noticed for this wiki that if they are not signed in to GitHub (or perhaps an admin on the repo) they are not able to edit. I like the idea of having a dynamic page that can be copied and pasted in before a release.

Then perhaps we should get back to a Google Doc :thinking: similar to the wiki page that Chris M created

**Comment by KirstieJane** [comment]
The wiki is currently set up to only allow edits from folks with push access. I’d recommend undoing that checkbox (in settings) and then only being logged into GitHub will be required (which I think is good because then we can track who makes changes).

Just while I’m here, I don’t think it should be the responsibility of the contributor to add themselves. It makes me sad that folks here have apologised and called themselves selfish while asking to be added, and I think there are likely a lot of other people who don’t put themselves forward when they HAVE contributed. In the Turing Way community we have a box in the PR template that contributors have to check to say that they’ve added themselves to the list which I think nudges recording the contribution well, but ultimately I think the person reviewing should check that they’re there.

**Comment by sappelhoff** [comment]
I think we can merge this PR for now, keeping the following three points in mind:

1. We should regularly ask the community to update / add their names to the contributors list ... some/one week/s before each release might be an acceptable time frame
1. We should use the WIKI page for contributors to "add themselves", then to be transferred into a commit via some maintainer (or contributor who wants to do it) --> we should make clear, that a GitHub account is needed for this
1. Whoever merges a pull request by some contributor should double check and assure that contributor is listed in the contributors list

**Comment by franklin-feingold** [comment]
This PR should be ready (added in Mikael from https://github.com/bids-standard/bids-specification/pull/292) 

+1 with the 3 points to keep in mind. Perhaps editing the [PR template](https://github.com/bids-standard/bids-specification/blob/master/pull_request_template.md) could be helpful for ensuring new contributors are properly credited for their time and work. My only concern with the wiki is the necessity for a GitHub account (could be a low barrier but something to consider)

**Comment by KirstieJane** [comment]
Are the three points @sappelhoff mentioned in the comment above https://github.com/bids-standard/bids-specification/pull/284#issuecomment-520253161 captured somewhere? Is there any sort of guidance for maintainers/reviewers that exists? 

If it doesn't, then I think "keeping things in mind" is a rather fragile solution.

**Comment by franklin-feingold** [comment]
I agree - I think documenting these points is needed, we have the [maintainers doc](https://github.com/bids-standard/bids-specification/issues/288) that will provide most of this guidance. I think 1 and 3 can be in the [maintainers guide](https://github.com/bids-standard/bids-specification/issues/288), 2 could be the [PR template](https://github.com/bids-standard/bids-specification/blob/master/pull_request_template.md). Noting it here to further place into a issue

wdyt?

**Comment by KirstieJane** [comment]
Yep! Those places sound great! Just wanted to make sure the information didn’t stay in a “soon to be closed” PR 🌟

**Comment by franklin-feingold** [comment]
cc @effigies for a review - I think this is ready to merge

**Comment by sappelhoff** [comment]
thanks @franklin-feingold et al.

**Pull request merged**